src/utilities/OpenWeather.tsx

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const getTemperature = (zipcode: string) => {
+  // Returns promise in which value contains the temperature or error
+  return axios.get('http://api.openweathermap.org/data/2.5/weather',
+  {params: {
+    zip: zipcode,
+    units: 'imperial',
+    appid: process.env.REACT_APP_OPENWEATHER_API_KEY}}
+  )
+  .then(function (response) {
+    // Returns temperature in fahrenheit as string
+    return JSON.stringify(response['data']['main']['temp']);
+  }) 
+  .catch(function (error) {
+    return error
+  })
+  
+}
+
+export default getTemperature;
```

Comment:
This file doesn't need the `.tsx` extension

Reply:
Changed .tsx to .ts